e2e/playwright/auth.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { expect, test } from '@e2e/playwright/zoo-test'
+
+// test file is for testing auth functionality
+test.describe('Authentication tests', () => {
+  test(
+    `The user can sign out and back in`,
+    { tag: ['@electron'] },
+    async ({ page, homePage, signInPage, toolbar, tronApp }) => {
+      if (!tronApp) {
+      }
```

Comment:
Other spots like this call `throwTronAppMissing()` or `fail()`. Do we need something similar here?

Reply:
Oh yeah good catch, must have deleted by mistake.

Reply:
Added `fail()`, ty!